.github/workflows/verify-py-update-notifier.yml

```diff
@@ -0,0 +1,35 @@
+name: Update verify.py
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  update_verify_py:
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]' && ${{ !github.event.pull_request.auto_merge }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.11"
+      - name: Run update_verify_py Script
+        env:
+          CHANGED_FILES: "${{ steps.changed-files.outputs.all_changed_files }}"
+        run: |
+          echo "==== $(date): Starting pipenv setup... ===="
+          python -m pip install --upgrade pip
+          pip install pipenv
+          pwd
+          cd "$GITHUB_WORKSPACE/.github/workflows/update-verify-py"
+          pipenv install
```

Comment:
we dont have any dependencies in that pipfile, can we just run on python and get rid of the environment?